app/views/profile/show.html.erb

```diff
@@ -34,6 +34,8 @@
       <p class="govuk-body"><span class="govuk-body govuk-!-font-weight-bold"><%= t 'profile.email' %></span><%= @user.email %></p>
       <p class="govuk-body"><span class="govuk-body govuk-!-font-weight-bold"><%= t 'profile.phone' %></span><%= @user.phone_number %></p>
       <p class="govuk-body"><span class="govuk-body govuk-!-font-weight-bold"><%= t 'profile.roles' %></span><%= @user.roles %></p>
+      <p class="govuk-body"><span class="govuk-body govuk-!-font-weight-bold"><%= t 'profile.team' %></span><%= @user.team %></p>
+      <p class="govuk-body"><span class="govuk-body govuk-!-font-weight-bold"><%= t 'profile.cognito_groups' %></span><%= @user.cognito_groups.join(', ') unless @user.team.nil? %></p>
```

Comment:
Do we want to show cognito groups as well as teams? This would probably confuse a general user. If we're keeping the profile page I guess we should limit showing cognito groups to gds_users